src/components/StoriesGroup/components/StoriesPreview/StoriesPreview.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {Icon} from '../../../Icon/Icon';
+import {MediaRenderer} from '../../../Stories/components';
+import {block} from '../../../utils/cn';
+import {StoriesGroupItem} from '../../types';
+import {Chevron} from '../../../icons/Chevron';
+import './StoriesPreview.scss';
+import {Button} from '../../../Button';
+
+const b = block('stories-group-preview');
+
+type StoriesPreviewListProps = {
+    groups: StoriesGroupItem[];
+    groupIndex: number;
+    onGroupSelect: (groupIndex: number) => void;
+    offset?: number;
+};
+const StoriesPreviewList = ({
+    groupIndex,
+    groups,
+    onGroupSelect,
+    offset = 0,
+}: StoriesPreviewListProps) => {
+    return (
+        <React.Fragment>
+            {groups.map(({previewMedia, items}, i) => {
+                const media = previewMedia || items[0]?.media;
+
+                return (
+                    <div
+                        key={i}
+                        className={b('stories-preview-item', {
+                            active: offset + i === groupIndex,
+                        })}
+                        onClick={() => onGroupSelect(offset + i)}
+                    >
+                        {media && <MediaRenderer media={media} />}
+                    </div>
+                );
+            })}
+        </React.Fragment>
+    );
+};
+
+type StoriesPreviewWithSliderProps = {
+    groups: StoriesGroupItem[];
+    groupIndex: number;
+    onGroupSelect: (groupIndex: number) => void;
+};
+const StoriesPreviewWithSlider = ({
+    groupIndex,
+    groups,
+    onGroupSelect,
+}: StoriesPreviewWithSliderProps) => {
+    const [offset, setOffset] = React.useState(0);
+
+    React.useEffect(() => {
+        const currentOffset = 12 * Math.floor(groupIndex / 12);
```

Comment:
12 is [magic number](https://eslint.org/docs/latest/rules/no-magic-numbers) here, lets make constant with this value or even make it property

Reply:
[eacd251](https://github.com/gravity-ui/uikit/pull/387/commits/eacd251693e128d07a54cafe132fef58e30d1f15)